src/Aggregate/FQCNAggregateFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+namespace MyOnlineStore\EventSourcing\Aggregate;
+
+use MyOnlineStore\EventSourcing\Event\Event;
+use MyOnlineStore\EventSourcing\Service\Assertion;
+
+final class FQCNAggregateFactory implements AggregateFactory
+{
+    /**
+     * @param Event[] $events
+     */
+    public function reconstituteFromHistory(
```

Comment:
This docblock is the same as the interface and may be omitted (this occurs in other classes as well)

Reply:
It's due to a limitation of code sniffer. It processes the code on a per-file base, so it doesn't know anything about the interface while processing the implementation. If this docblock is removed, the codestyle check fails.

Reply:
:disappointed: okay